src/components/stepper/icon-stepper-subtracting.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";

```

Comment:
porque um ta em componente react e o outro em svg? seria interessante manter o mesmo padrão para os dois, outra coisa que recomendo é ter uma pasta chamada assets para colocar todos icones e imagens, organiza melhor o código dessa forma

Reply:
Bom, um eu precisava trocar a cor dele quando o valor for zero ai fiz um component e passei props pra ele, e os outros eu não precisava passar props. 

vou criar a pasta e colocar eles em um compoent 👍